### DISCUSSION TITLE
need to get streamurl

### DISCUSSION THREAD

**Discussion by Amalraj-Koovat** [ant-media/Ant-Media-Server#6026] [Q&A]
how i can get streamurl of a livestream? currently when i do api call streamurl is null

**Comment by USAMAWIZARD** [comment]
@Amalraj-Koovat can you please clarify which URL are you looking for ? do you want to embed the stream ? or are you looking for the HLS URL ?

**Comment by Amalraj-Koovat** [comment]
want to play the live stream using the url. there is a stream url in the response to the rest api http://localhost:5080/LiveApp/rest/v2/broadcasts/{streamID} . but it returns null. also i want to know is there any api which can stream the live stram data (like buffers) from the ant media server community edition runs on port 5080 to a local node server running on some other port or to a frontend framework like react

**Reply by USAMAWIZARD** [reply]
Can you please explain why you are trying to pass the video buffer through node ? you don't need to pass the video buffer through node you can directly play the HLS on the client side , here are the examples for that.

https://antmedia.io/docs/guides/playing-live-stream/embedded-web-player/

**Comment by Amalraj-Koovat** [comment]
using react as frontend. ant media server is running in my localhost. react is running from another machine in the same network. trying to access to my machine through my ip address like http://{my_ip_address}:5080/LiveApp/play.html?id=12345. but the video is not playing and not showing any message on the screen. tried using iframe also. not working still . please help me

**Reply by USAMAWIZARD** [reply]
Hi @Amalraj-Koovat  , Can you try loading this on same machine which is running ant media and see if works ?
